PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -268,29 +319,29 @@ bool FemtoDreamTrackSelection::isSelectedMinimal(T const& track)
   }
 
   /// check whether the most open cuts are fulfilled
-  const static int nPtMinSel = getNSelections(femtoDreamTrackSelection::kpTMin);
-  const static int nPtMaxSel = getNSelections(femtoDreamTrackSelection::kpTMax);
-  const static int nEtaSel = getNSelections(femtoDreamTrackSelection::kEtaMax);
-  const static int nTPCnMinSel = getNSelections(femtoDreamTrackSelection::kTPCnClsMin);
-  const static int nTPCfMinSel = getNSelections(femtoDreamTrackSelection::kTPCfClsMin);
-  const static int nTPCcMinSel = getNSelections(femtoDreamTrackSelection::kTPCcRowsMin);
-  const static int nTPCsMaxSel = getNSelections(femtoDreamTrackSelection::kTPCsClsMax);
-  const static int nDCAxyMaxSel = getNSelections(femtoDreamTrackSelection::kDCAxyMax);
-  const static int nDCAzMaxSel = getNSelections(femtoDreamTrackSelection::kDCAzMax);
-  const static int nDCAMinSel = getNSelections(femtoDreamTrackSelection::kDCAMin);
-  const static int nPIDnSigmaSel = getNSelections(femtoDreamTrackSelection::kPIDnSigmaMax);
-
-  const static float pTMin = getMinimalSelection(femtoDreamTrackSelection::kpTMin, femtoDreamSelection::kLowerLimit);
-  const static float pTMax = getMinimalSelection(femtoDreamTrackSelection::kpTMax, femtoDreamSelection::kUpperLimit);
-  const static float etaMax = getMinimalSelection(femtoDreamTrackSelection::kEtaMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float nClsMin = getMinimalSelection(femtoDreamTrackSelection::kTPCnClsMin, femtoDreamSelection::kLowerLimit);
-  const static float fClsMin = getMinimalSelection(femtoDreamTrackSelection::kTPCfClsMin, femtoDreamSelection::kLowerLimit);
-  const static float cTPCMin = getMinimalSelection(femtoDreamTrackSelection::kTPCcRowsMin, femtoDreamSelection::kLowerLimit);
-  const static float sTPCMax = getMinimalSelection(femtoDreamTrackSelection::kTPCsClsMax, femtoDreamSelection::kUpperLimit);
-  const static float dcaXYMax = getMinimalSelection(femtoDreamTrackSelection::kDCAxyMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float dcaZMax = getMinimalSelection(femtoDreamTrackSelection::kDCAzMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float dcaMin = getMinimalSelection(femtoDreamTrackSelection::kDCAMin, femtoDreamSelection::kAbsLowerLimit);
-  const static float nSigmaPIDMax = getMinimalSelection(femtoDreamTrackSelection::kPIDnSigmaMax, femtoDreamSelection::kAbsUpperLimit);
+  nPtMinSel = getNSelections(femtoDreamTrackSelection::kpTMin);
```

Comment:
I am confused. Why here the cut value is here still set on **each** track (this will still be slow). You should do that only once somewhere, e.g. in init.